src/constants.py

```diff
@@ -7,8 +7,8 @@
 another module or to resolve circular imports.
 """
 DEFAULT_BRANCH = "main"
-DOCUMENTATION_TAG = "upload-charm-docs/base-content"
-DISCOURSE_AHEAD_TAG = "upload-charm-docs/discourse-ahead-ok"
+DOCUMENTATION_TAG = "discourse-gatekeeper/base-content"
+DISCOURSE_AHEAD_TAG = "discourse-gatekeeper/discourse-ahead-ok"
```

Comment:
I believe this would be a breaking change, probably something to note for repositories already using it!

Reply:
Not really, see [my comment above](https://github.com/canonical/upload-charm-docs/pull/210#issuecomment-1709627616)